ocs_ci/ocs/platform_nodes.py

```diff
@@ -312,6 +312,21 @@ def restart_nodes(self, nodes, wait=True, force=True):
         )
         self.aws.restart_ec2_instances(instances=instances, wait=wait, force=force)
 
+    def terminate_nodes(self, nodes, wait=True):
```

Comment:
Will this method be relevant to other platforms? If so, please add a stub method to NodesBase and VMWareNodes

Reply:
No. This function is specific to only AWS.